source/includes/properties/_01-create.md

```diff
@@ -0,0 +1,11 @@
+## Create
+
+```shell
+curl -d "value=PropertyValue" http://goserver.com:8153/go/properties/PipelineName/1243/StageName/1/JobName/MyPropertyName
```

Comment:
Also please add `-X POST`, `-d` implies `-X POST` but since we're documenting the API, it's good to indicate that it is supposed to be a POST.